OPHD/Things/Structures/OreRefining.h

```diff
@@ -28,4 +28,6 @@ class OreRefining : public Structure
 
 protected:
 	virtual int calculateMaxStorage() = 0;
+
+	std::array<int, 4> OreConversionDivisor{ 2, 2, 3, 3 };
```

Comment:
The idea here was to allow for tweaking in derived classes but this change still makes sense as it provides a base conversion rate that can still be overridden in classes inheriting from OreRefining. Commenting on this mostly for the sake of documenting design decisions.